frontend/src/index.html

```diff
@@ -1,13 +1,14 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>Frontend</title>
-  <base href="/">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
-</head>
-<body>
-  <app-root class="flex col"></app-root>
-</body>
+  <head>
+    <meta charset="utf-8" />
+    <title>Frontend</title>
+    <base href="/" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDYJ2elwHMpMisrji9VBvbYHsMOs3Lsprg"></script>
```

Comment:
can you just add the script somewhere else where you can inject the apikey?

Reply:
DONE